chain/chunks/src/lib.rs

```diff
@@ -1699,17 +1699,14 @@ impl ShardsManager {
 
 #[cfg(test)]
 mod test {
-    use crate::test_utils::{ChunkForwardingTestFixture, SealsManagerTestFixture};
-    use crate::{
-        ChunkRequestInfo, ProcessPartialEncodedChunkResult, Seal, SealsManager, ShardsManager,
-        CHUNK_REQUEST_RETRY_MS, NUM_PARTS_REQUESTED_IN_SEAL,
-    };
+    use super::*;
+    use crate::test_utils::*;
     use near_chain::test_utils::KeyValueRuntime;
     use near_network::test_utils::MockNetworkAdapter;
     #[cfg(feature = "protocol_feature_forward_chunk_parts")]
     use near_network::types::PartialEncodedChunkForwardMsg;
     use near_primitives::hash::{hash, CryptoHash};
-    use near_primitives::sharding::{ChunkHash, PartialEncodedChunkV2};
+    #[cfg(feature = "protocol_feature_forward_chunk_parts")]
```

Comment:
this is kinda weird. shouldn't this be guarded by `nightly_protocol` instead?

Reply:
My thinking about this is features and `nightly_protocol` should be independent. Because if you want to release the feature, you'd just have to move it to `stable_protocol`. If any of the tests rely on `nightly_protocol` feature, then they will break